src/corehost/cli/runtime_config.cpp

```diff
@@ -112,3 +112,18 @@ bool runtime_config_t::get_portable() const
 {
     return m_portable;
 }
+
+void runtime_config_t::config_kv(std::vector<std::string>* keys, std::vector<std::string>* values) const
+{
+    for (const auto& kv : m_properties)
+    {
+        keys->push_back(pal::to_stdstring(kv.first));
+        values->push_back(pal::to_stdstring(kv.second));
+    }
+
+    if (m_properties.count(_STRINGIFY(System_GC_Server)) == 0)
```

Comment:
@gkhanna79 The host doesn't need to know about properties that are added. It can just pass the strings as they are unless (as in this case) it wants to explicitly force something if it wasn't set.

I don't know the details of that request, but if we really want to change the default behavior of CoreCLR  when server GC is not set explicitly, the place to do that would be in the definition of the CLRConfigValue for it (in clrconfigvalues.h). Once this system is in place, we will fall back on the defaults specified there if something is not set (for now, since we're respecting both methods of configuration for now).